### ISSUE TITLE
[QUESTION] How to use ssh-rsa/ed25519 keys?

### ISSUE THREAD

**Issue by thenoseman** [slok/agebox#61]
Hello!

I'm evaluating `agebox` as an alternative to the currently used "blackbox" scripts.

This is where I found that I can *encrypt* files using my ed25519 based ssh key but the decryption fails. I have tested this on MacOS (10.15.7) and Linux:

```
> mkdir agebox-test
> cd agebox-test
> echo "HELLO" > testfile
> agebox init

> mkdir keys
> ssh-keygen -t ed25519 -N "passphrase"
# copy id_es25519.pub to ./keys

> agebox encrypt testfile
INFO[0000] Loaded public keys                            keys=1 svc=storage.fs.KeyRepository version=v0.2.0
INFO[0000] Secret encrypted                              secret-id=testfile svc=box.encrypt.Service version=v0.2.0
> agebox decrypt testfile.agebox --private-key id_es25519 # private key file is present in this directory

error: "decrypt" command failed: could not decrypt: could not get private key: could not load private key in "id_es25519": invalid private key
```

Can you help me finding out what I do wrong?

Thank you for building agebox and for you help
Frank

**Comment by slok** [comment]
Hi @thenoseman!

Thanks for the report! tThe error is caused because Agebox doesn't support SSH keys with a passphrase. However this week I plan to support them and publish a new release along with the `validate` command that I added yesterday.

**Comment by slok** [comment]
Just released [v0.3.0](https://github.com/slok/agebox/releases/tag/v0.3.0). This adds support for passphrases, so it should fix your problem.

You can also pass a passphrase using flags in case you want to disable interactive mode:
```
agebox decrypt testfile.agebox --private-key id_es25519 --passphrase passphrase
```

**Comment by thenoseman** [comment]
That was fast! Many thanks!
